src/custom-elements/src/components/spinner/spinner.tsx

```diff
@@ -0,0 +1,75 @@
+import { Component, Prop, Watch } from '@stencil/core';
+import { ICON_SIZES } from '../../constants/size';
+
+const SPINNER_COLORS = ['', 'primary', 'success', 'warning', 'danger', 'muted', 'secondary', 'light'];
```

Comment:
@jllr how can we be more consistent with our colors? We have the following list for colors: 
export const TEXT_COLORS = ['', 'primary', 'success', 'warning', 'danger', 'muted', 'inverse'];
export const SEMANTIC_COLORS = ['info', 'warning', 'danger', 'success'];
const SPINNER_COLORS = ['', 'primary', 'success', 'warning', 'danger', 'muted', 'secondary', 'light'];

Can we unify in any way? Thank you!